### ISSUE TITLE
Specified x and y background repeat properties are ignored

### ISSUE THREAD

**Issue by DingoEatingFuzz** [jakubpawlowicz/clean-css#453]
To reproduce:

File `test.css`
```css
.broken .thing {
	background: url('/image/path.png') no-repeat repeat 0 0 transparent;
}
```

Running
```sh
$ cleancss test.css
```

Results in
```css
.broken .thing{background:url(/image/path.png) no-repeat}
```

Expected
```css
.broken .thing{background:url(/image/path.png) no-repeat repeat}
```

Removing the `0 0 transparent`, in this case, is fine, but `no-repeat` is different than `no-repeat repeat`.

**Comment by jakubpawlowicz** [comment]
It's a bug indeed. Please use `--skip-shorthand-compacting` for the time being!

**Comment by DingoEatingFuzz** [comment]
Thanks!

**Comment by jakubpawlowicz** [comment]
Fixed in 3.0.10.
